client/components/Nav.jsx

```diff
@@ -244,87 +257,89 @@ class Nav extends React.PureComponent {
 
   renderProjectMenu(navDropdownState) {
     return (
-      <ul className="nav__items-left">
-        <li className="nav__item-logo">
-          <LogoIcon role="img" aria-label="p5.js Logo" focusable="false" className="svg__logo" />
-        </li>
-        <li className={navDropdownState.file}>
-          <button
-            onClick={this.toggleDropdownForFile}
-            onBlur={this.handleBlur}
-            onFocus={this.clearHideTimeout}
-            onMouseOver={() => {
-              if (this.state.dropdownOpen !== 'none') {
-                this.setDropdown('file');
-              }
-            }}
-          >
-            <span className="nav__item-header">File</span>
-            <TriangleIcon className="nav__item-header-triangle" focusable="false" aria-hidden="true" />
-          </button>
-          <ul className="nav__dropdown">
-            <li className="nav__dropdown-item">
-              <button
-                onClick={this.handleNew}
-                onFocus={this.handleFocusForFile}
-                onBlur={this.handleBlur}
-              >
-                New
-              </button>
+      <Translation ns="Menu">
```

Comment:
What do you think about using the [`withTranslation` HoC](https://react.i18next.com/latest/withtranslation-hoc) instead of this `Translation` component?

I think it would be clearer to see what's actually being changed in these files as we'd wrap Nav in `withTranslation('Menu')(Nav)` at the bottom of the file and then use `this.props.t('key')`.

This would avoid indenting all the files and would be less changes in the PR diff views.